README.md

```diff
@@ -1,118 +1,127 @@
-# opa-istio-plugin
+# opa-envoy-plugin
 
-[![Build Status](https://travis-ci.org/open-policy-agent/opa-istio-plugin.svg?branch=master)](https://travis-ci.org/open-policy-agent/opa-istio-plugin) [![Go Report Card](https://goreportcard.com/badge/github.com/open-policy-agent/opa-istio-plugin)](https://goreportcard.com/report/github.com/open-policy-agent/opa-istio-plugin)
+[![Build Status](https://travis-ci.org/open-policy-agent/opa-envoy-plugin.svg?branch=master)](https://travis-ci.org/open-policy-agent/opa-envoy-plugin) [![Go Report Card](https://goreportcard.com/badge/github.com/open-policy-agent/opa-envoy-plugin)](https://goreportcard.com/report/github.com/open-policy-agent/opa-envoy-plugin)
 
-This repository contains an extended version of OPA (**OPA-Istio**) that allows you to enforce OPA
-policies at the Istio Proxy layer.
+This repository contains an extended version of OPA (**OPA-Envoy**) that allows you to enforce OPA
+policies at the Envoy Proxy layer.
 
 ## Issue Management
 Use [OPA GitHub Issues](https://github.com/open-policy-agent/opa/issues) to request features or file bugs.
 
 ## Overview
 
-OPA-Istio extends OPA with a gRPC server that implements the [Envoy External
+OPA-Envoy extends OPA with a gRPC server that implements the [Envoy External
 Authorization
 API](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/ext_authz_filter.html).
 You can use this version of OPA to enforce fine-grained, context-aware access
-control policies at the Istio Proxy layer without modifying your microservice.
+control policies at the Envoy Proxy layer without modifying your microservice.
 
 ## How does it work?
 
-In addition to the Istio Proxy/Envoy sidecar, your application pods will include an OPA
-sidecar. When Istio Proxy receives API requests destined for your
+In addition to the Envoy sidecar, your application pods will include an OPA
+sidecar. When Envoy receives API requests destined for your
 microservice, it checks with OPA to decide if the request should be allowed.
 
-Evaluating policies locally at the Istio Proxy layer is preferable because it
+Evaluating policies locally at the Envoy Proxy layer is preferable because it
 avoids introducing a network hop (which has implications on performance and
 availability) in order to perform the authorization check.
```

Comment:
Again, Envoy Proxy layer should go. Also, let's include a note that OPA-Envoy can be used outside of Kubernetes environments as well. The examples below all refer to pods and show how to do it in Kubernetes, but there's nothing stopping someone from deploying OPAs next to Envoys outside of Kubernetes.

Reply:
Done and added a note.